components/Tabs/Tabs.js

```diff
@@ -2,71 +2,77 @@ class TabLink {
-    // this.cards = Array.from(this.cards).map();
+     // Map over the newly converted NodeList we just created in our if statement above.
+     // Convert each this.cards element into a new instance of the TabCard class. Pass in a card object to the TabCard class.
+    this.cards = Array.from(this.cards).map(card => new TabCard(card));
 
     // Add a click event that invokes this.selectTab
     // this.tabElement.addEventListener();
+    this.tabElement.addEventListener('click', () => this.selectTab())
   }
 
   selectTab(){
 
     // Select all elements with the .tab class on them
-    // const tabs = document.querySelectorAll();
-    
+    const tabs = document.querySelectorAll('.tab');
+
     // Iterate through the NodeList removing the .active-tab class from each element
-    // tabs.forEach()
+    tabs.forEach(tab => tab.classList.remove('active-tab'));
 
     // Select all of the elements with the .card class on them
-    // const cards = ;
+    const cards = document.querySelectorAll('.card');
 
     // Iterate through the NodeList setting the display style each one to 'none'
-    // cards.forEach()
-    
+    cards.forEach(card => card.style.display = 'none');
+
     // Add a class of ".active-tab" to this.tabElement
-    // this.tabElement;
-  
-    // Notice we are looping through the this.cards array and invoking selectCard() from the TabCard class. Just un-comment the code and study what is happening here.
-    // this.cards.forEach(card => card.selectCard());
+    this.tabElement.classList.add('active-tab');
+    // Notice we are looping through the this.cards array and invoking selectCard() from the TabCard class.
+    //Just un-comment the code and study what is happening here.
+    this.cards.forEach(card => card.selectCard());
   }
 }
 
 class TabCard {
   constructor(cardElement){
     // Assign this.cardElement to the cardElement DOM reference
-    // this.cardElement;
+    this.cardElement = cardElement;
   }
   selectCard(){
     // Update the style of this.cardElement to display = "flex"
     // this.cardElement;
+    this.cardElement.style.display = "flex";
   }
 
 }
 
-/* START HERE: 
+///* START HERE:
+
+//- Select all classes named ".tab" and assign that value to the tabs variable
+
+//- With your selection in place, now chain a .forEach() method onto the tabs variable to iterate over the DOM NodeList
+
+//- In your .forEach() method's callback function, return a new instance of TabLink and pass in each tab as a parameter
 
-- Select all classes named ".tab" and assign that value to the tabs variable
 
-- With your selection in place, now chain a .forEach() method onto the tabs variable to iterate over the DOM NodeList
 
-- In your .forEach() method's callback function, return a new instance of TabLink and pass in each tab as a parameter
+let tabs = document.querySelectorAll('.tab');
 
-*/
-let tabs = document.querySelectorAll();
\ No newline at end of file
+tabs.forEach(tab => new TabLink(tab));
```

Comment:
What you did id perfect, but here is how you can do it in one line:

let tabs = document.querySelectorAll('.tab').forEach(tab => new TabLink(tab))